miner/miner.go

```diff
@@ -95,6 +104,16 @@ func New(eth Backend, config *Config, chainConfig *params.ChainConfig, mux *even
 		stopCh:  make(chan struct{}),
 		worker:  newWorker(config, chainConfig, engine, eth, mux, isLocalBlock, false),
 	}
+	antsPool, err := ants.NewPool(MevRoutineLimit)
```

Comment:
it is strange `MevRoutineLimit` is defined in another file but only used in this file.

10000 is far too large a setting. 50 is acceptable.

Reply:
deleted antspool